templates/appliance/plex.html

```diff
@@ -0,0 +1,171 @@
+{% extends "appliance/_base-appliance.html" %}
+
+{% block title %}Plex Media Server Ubuntu Appliance{% endblock %}
+
+{% block meta_description %}{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1zgOl4GCK6O6Clv6oxp-DXDSKYIKO2pvpXaYXALiTlpo/edit#heading=h.c6wjbvvp8mcm{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <div class="p-media-object u-no-margin--bottom">
+        <img src="https://assets.ubuntu.com/v1/402069bd-plex-media-server.png?w=75" class="p-media-object__image is-round" height="75" width="75">
+        <div class="p-media-object__details">
+          <h1>
+            Plex Media Server
+          </h1>
+          <p class="p-heading--three">Ubuntu Appliance</p>
+          <ul class="p-inline-list--middot">
+            <li class="p-inline-list__item">By Plex, Inc.</li>
+            <li class="p-inline-list__item">Entertainment</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 u-align--right">
+      <form class="p-form--inline js-download">
+        <div class="p-form__group">
+          <label for="platform" class="u-off-screen">Platform</label>
+          <select id="platform" class="js-platform-select">
+            <option value="rpi">Raspberry Pi</option>
+            <option value="kvm">KVM</option>
+            <option value="intel-nuc">Intel NUC</option>
+          </select>
+        </div>
+        <div class="p-form__group">
+          <a href="/appliance/plex/rpi" class="p-button--positive p-form__control js-download-link">Download</a>
```

Comment:
The copy docs say this should like to /appliance/plex/thank-you

Reply:
There's no link stated in the copydoc. The new flow they introduced yesterday has different pages for each platform's download page

Reply:
Did that change from stand up yesterday?

Reply:
We found out about it at the 10:30 sync yesterday